build/ci/evergreen.yml

```diff
@@ -902,8 +902,56 @@ tasks:
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud-manager
           E2E_TAGS: cloudmanager,generic
-  - name: ops_manager_generic_4_4_e2e
-    tags: ["opsmanager", "general"]
+  # Deploy a replica set, we need to be careful on running parallel modifications to the automation config
+  - name: cloud_manager_deploy_replica_set_e2e
```

Comment:
couple of things in this file, grouping ops manager tests by ops manager version, adding OM5.0 removing OM4.2 (EOL)